opentelemetry-instrumentation/tests/test_run.py

```diff
@@ -107,18 +112,22 @@ class TestArgs(TestCase):
     def test_exporter(self, _):  # pylint: disable=no-self-use
         with patch("sys.argv", ["instrument", "2"]):
             auto_instrumentation.run()
-            self.assertIsNone(environ.get("OTEL_EXPORTER"))
+            self.assertIsNone(environ.get(OTEL_METRICS_EXPORTER))
 
-        with patch("sys.argv", ["instrument", "-e", "zipkin", "1", "2"]):
+        with patch(
+            "sys.argv", ["instrument", "--trace-exporter", "jaeger", "1", "2"]
+        ):
             auto_instrumentation.run()
-            self.assertEqual(environ.get("OTEL_EXPORTER"), "zipkin")
+            self.assertEqual(environ.get(OTEL_TRACE_EXPORTER), "jaeger")
 
     @patch("opentelemetry.instrumentation.auto_instrumentation.execl")
     def test_service_name(self, _):  # pylint: disable=no-self-use
         with patch("sys.argv", ["instrument", "2"]):
             auto_instrumentation.run()
-            self.assertIsNone(environ.get("OTEL_SERVICE_NAME"))
+            self.assertIsNone(environ.get(OTEL_PYTHON_SERVICE_NAME))
 
         with patch("sys.argv", ["instrument", "-s", "my-service", "1", "2"]):
             auto_instrumentation.run()
-            self.assertEqual(environ.get("OTEL_SERVICE_NAME"), "my-service")
+            self.assertEqual(
+                environ.get(OTEL_PYTHON_SERVICE_NAME), "my-service"
```

Comment:
Probably not in this PR scope. I believe this is used for exporter `service_name` param. Shouldn't we remove this env and service_name arg and rely on `Resource` attributes for service name.